data.js

```diff
@@ -528,6 +528,21 @@ module.exports = [
       'Productivity',
     ],
   },
+  {
+    name: 'Umair Hafeez',
+    description: 'Front End Architect & Angular Fanatic 🅰️. I share whatever I have learned or learn during my day to day job. I believe in lean, share and grow philosophy, and that\'s the reason I started writing on this blog along with byte sized tips I share on Twitter and LinkedIn.',
+    url: 'https://blog.umairhafeez.com',
+    twitter: '@_UmairHafeez_',
+    tags: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+      'TypeScript',
+      'Angular',
+      'Web Development',
+      'Web Accessibility'
```

Comment:
Could you change 'Web Accessibility' to 'Accessibility'? There's another person using 'Accessibility', so we can align :)

Reply:
@Dawntraoz I've updated as per your comment 🙂